docs/r2/tensorboard_in_notebooks.ipynb

```diff
@@ -77,7 +77,14 @@
       },
       "cell_type": "markdown",
       "source": [
-        "TensorBoard can be used directly within notebook experiences such as [Colab](https://colab.research.google.com/) and [Jupyter](https://jupyter.org/). This can be helpful for sharing results, integrating TensorBoard into existing workflows, and using TensorBoard without installing anything locally."
+        "TensorBoard can be used directly within notebook experiences such as [Colab](https://colab.research.google.com/) and [Jupyter](https://jupyter.org/). This can be helpful for sharing results, integrating TensorBoard into existing workflows, and using TensorBoard without installing anything locally.\n",
```

Comment:
Would it be better to move this new text to the "setup" section below?

Reply:
Sure, can do.